lib/pdk/util.rb

```diff
@@ -8,6 +8,14 @@
 
 module PDK
   module Util
+    PACKAGED_TEMPLATE_KEYWORD = 'pdk-default'.freeze
+
+    PACKAGED_TEMPLATE_PATHS = {
+      'windows' => 'file://C:/Program Files/Puppet Labs/DevelopmentKit/share/cache/pdk-templates.git',
```

Comment:
Reminder to mention "non-default install locations on Windows need manual intervention by the user to fix this" in the Release Notes.

Reply:
We currently don't support changing the install location on Windows, so we know that anything in a version of PDK prior to the release of this fix will be at that static file path. Any modules converted or created after the introduction of this fix will appropriately show the static string.